app/pb_kits/playbook/pb_pill/_pill.scss

```diff
@@ -0,0 +1,33 @@
+@import "../tokens/spacing";
+@import "../tokens/colors";
+@import "../tokens/typography";
+@import "../tokens/opacity";
+
+$pb_pill_height: 22px;
+
+[class^=pb_pill] {
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  padding: 0 $space-sm/1.8;
+  height: $pb_pill_height;
+  border-radius: $pb_pill_height/2;
+
+  [class^=pb_title] {
+    &[class*=_4] {
+      font-size: $text-small !important;
```

Comment:
Wondering how we treat these types of things. This size doesn't exist as a text pattern...but is part of our old variables. However, should this just use base? I'm conflicted because it looks good. In any case...we should clean up this file: https://github.com/powerhome/playbook/blob/master/app/pb_kits/playbook/tokens/_typography.scss